src/Interactive/Host/Interactive/Core/InteractiveHost.LazyRemoteService.cs

```diff
@@ -73,10 +73,7 @@ private async Task<InitializedRemoteService> TryStartAndInitializeProcessAsync(C
                     });
 
                     // try to execute initialization script:
-                    var initializationResult = await Async<RemoteExecutionResult>(remoteService, (service, operation) =>
-                    {
-                        service.InitializeContext(operation, Options.InitializationFile, isRestarting: InstanceId > 1);
-                    }).ConfigureAwait(false);
+                    var initializationResult = await Host.Async<RemoteExecutionResult>("InitializeContext", Options.InitializationFile, InstanceId > 1).ConfigureAwait(false);
```

Comment:
>Host [](start = 53, length = 4)

There were two `Async` methods on the InteractiveHost - one instance and one static. We need to call the static one here since that one takes an already created `remoteService`. 
If you called the instance one it will try to initialize the service again, which will call here and loop. We don't want that.